web/.storybook/main.ts

```diff
@@ -63,7 +63,8 @@ const config: StorybookConfig = {
       if (plugin && 'name' in plugin) {
         return (
           plugin.name !== 'teleport-html-plugin' &&
-          plugin.name !== 'teleport-transform-html-plugin'
+          plugin.name !== 'teleport-transform-html-plugin' &&
+          plugin.name !== 'storybook:react-docgen-plugin'
```

Comment:
We don't have to disable this plugin. It didn't work because Storybook still uses babel for the Vite framework and our babel config wasn't fully configured.
After I added typescript preset to it, the errors gone away (I have it in the patch above).